bqrole/project.go

```diff
@@ -66,6 +67,49 @@ func PermitProject(role, project string, users []string, yes bool) error {
 	return nil
 }
 
+func RevokeProject(role, project string, users []string, yes bool) error {
+	ctx := context.Background()
+	client, err := bq.NewClient(ctx, project)
+	if err != nil {
+		return errors.New("failed to create bigquery Client")
+	}
+
+	fmt.Printf("REVOKE following PROJECT-WIDE permission\n")
+	fmt.Printf("project_id: %s\n", project)
+	fmt.Printf("role:       %s\n", role)
+	fmt.Printf("users:      %s\n", users)
+
+	if !yes {
+		fmt.Printf("If you proceeds, PROJECT-WIDE permission will be added. Are you sure? [y/n]")
+
+		reader := bufio.NewReader(os.Stdin)
+		res, err := reader.ReadString('\n')
+
+		if err != nil || strings.TrimSpace(res) != "y" {
+			fmt.Println("Abort.")
+			return nil
+		}
+	}
+
+	defer client.Close()
```

Comment:
I think client.Close should be called immediately

```go
	client, err := bq.NewClient(ctx, project)
	if err != nil {
		return errors.New("failed to create bigquery Client")
	}
	defer client.Close()
```

Reply:
Thanks! fixed